pkg/provisioner/ironic/ironic.go

```diff
@@ -1612,7 +1612,7 @@ func (p *ironicProvisioner) Deprovision(force bool) (result provisioner.Result,
 // Delete removes the host from the provisioning system. It may be
 // called multiple times, and should return true for its dirty flag
 // until the deprovisioning operation is completed.
-func (p *ironicProvisioner) Delete() (result provisioner.Result, err error) {
+func (p *ironicProvisioner) Delete(force bool) (result provisioner.Result, err error) {
```

Comment:
That's just a `forceHardPowerOff` I guess, otherwise the meaning looks too much generic in respect of the current implementation